dojo/management/commands/run_scan.py

```diff
@@ -99,10 +100,10 @@ def runIPScan(host, service_dict):
             try:
                 nm = PortScannerAsync()
             except PortScannerError:
-                print('Nmap not found', sys.exc_info()[0])
+                print(('Nmap not found', sys.exc_info()[0]))
```

Comment:
I think that here is print a tuple

Reply:
You are correct.  Lines 204 and 212 contain legacy __print__ statements so we must assume that this file has not been ported to Python 3.  In Python 2, this line (without the import print_function) would print-a-tuple so now when we add the import print_function, we must ensure that it prints-a-tuple on both Python 2 and on Python 3.

Reply:
Oh yes, I was confused. Sorry for the noise. 

IMO this print should be just a str not a tuple. But well, that is a core decision